src/consumers/kafkaStreamConsumer.js

```diff
@@ -19,7 +24,19 @@ module.exports = class KafkaStreamConsumer {
                 fetchMaxBytes: FetchMaxBytes || 1024 * 1024,
                 autoCommitIntervalMs: AutoCommitIntervalMs || 5000
             };
-
+            if (shouldExposeMetrics) {
+                this.kafkaQueryHistogram = new prometheus.Histogram({
```

Comment:
a. It's wrapped :)
b. the other consumer is not really built to run with the metrics. This is because we just invoking the users function and we don't really know whats going on afterwards